index.js

```diff
@@ -107,7 +107,7 @@ if (PNGImages) {
       verbose: !argv.mute,
       silent: false,
     }))
-    .pipe(rename({ prefix: argv.prefix, suffix: `${argv.suffix}.png`, extname: '.webp' }))
+    .pipe(rename({ prefix: argv.prefix, suffix: `${argv.suffix}`, extname: '.webp' }))
```

Comment:
Unnecessary string interpolation could be removed

Reply:
That's a good point, I'll remove it and take a look at the test later and re-commit. Thanks